tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -97,6 +97,80 @@ def test_raises_generic_errors(self, mock_logger):
                 )
 
 
+class TestBackSlashDeriveCodeMappings(BaseDeriveCodeMappings):
+    def setUp(self):
+        super().setUp()
+        self.platform = "python"
+        self.event_data = self.generate_data(
+            [
+                {"in_app": True, "filename": "\\sentry\\mouse.py"},
+                {"in_app": True, "filename": "\\sentry\\dog\\cat\\parrot.py"},
+                {"in_app": True, "filename": "C:sentry\\tasks.py"},
```

Comment:
```suggestion
                {"in_app": True, "filename": "C:\\sentry\\tasks.py"},
```
I think this is more accurate to reality.

Reply:
This distinction (to my surprise as well) actually exists within Windows, see [here](https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats) in the official Windows docs. TL;DR is that no backslash after the drive letter signals that it is a relative path

Reply:
Add a note and keep it, however, it is unlikely to happen on a stack trace.